src/ngraph/op/fused/group_conv.hpp

```diff
@@ -85,5 +85,83 @@ namespace ngraph
         private:
             bool has_groups_in_filters_shape() const;
         };
+
+        /// \brief Group Convolution data batch backprop
+        class GroupConvolutionBackpropData : public ngraph::op::util::FusedOp
+        {
+        public:
+            NGRAPH_API
+            static constexpr NodeTypeInfo type_info{"GroupConvolutionBackpropData", 0};
+            const NodeTypeInfo& get_type_info() const override { return type_info; }
+            GroupConvolutionBackpropData() = default;
+            GroupConvolutionBackpropData(const Output<Node>& data_batch,
+                                         const Output<Node>& filters,
+                                         const Output<Node>& output_delta,
+                                         const Strides& window_movement_strides,
+                                         const Strides& window_dilation_strides,
+                                         const CoordinateDiff& padding_below,
+                                         const CoordinateDiff& padding_above,
+                                         const size_t groups);
+
+            const Strides& get_window_movement_strides() const { return m_window_movement_strides; }
+            const Strides& get_window_dilation_strides() const { return m_window_dilation_strides; }
+            const CoordinateDiff& get_padding_below() const { return m_padding_below; }
+            const CoordinateDiff& get_padding_above() const { return m_padding_above; }
+            Output<Node> get_filters() { return input_value(1); }
+            Output<Node> get_data_batch() { return input_value(0); }
+            size_t get_groups() const;
+            virtual std::shared_ptr<Node>
+                copy_with_new_args(const NodeVector& new_args) const override;
+
+            virtual NodeVector decompose_op() const override;
+
+            virtual void pre_validate_and_infer_types() override;
+
+        protected:
+            Strides m_window_movement_strides;
+            Strides m_window_dilation_strides;
+            CoordinateDiff m_padding_below;
+            CoordinateDiff m_padding_above;
+            Dimension m_groups;
+        };
+
+        /// \brief Group Convolution filters backprop
+        class GroupConvolutionBackpropFilters : public ngraph::op::util::FusedOp
+        {
+        public:
+            NGRAPH_API
```

Comment:
The `NGRAPH_API` also needs to be moved.

Reply:
updated